src/defaultStore/state.js

```diff
@@ -1,9 +1,15 @@
-export default function(options) {
-
-    if (typeof window.__ENV__ === 'undefined') {
-        window.__ENV__ = 'dummy';
+function getDefault(key, defaultValue) {    
+    if (typeof window.__ENV__.workbench_data[key] !== 'undefined') {
```

Comment:
Cool 🌟 
Why not combine the null and undefined check in 1 statement?

Reply:
Yes think this whole thing can be done better